io.github.yuxshao.ptcollab.yml

```diff
@@ -0,0 +1,27 @@
+id: io.github.yuxshao.ptcollab
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+command: ptcollab
+rename-icon: ptcollab
+rename-desktop-file: ptcollab.desktop
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --device=dri
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --socket=fallback-x11
+  - --filesystem=home
+modules:
+  - name: ptcollab
+    buildsystem: qmake
+    build-commands:
```

Comment:
```suggestion
```

maybe that the failure you don't describe.

Reply:
Oh yeah, i thought i pushed that already